pages/authzed/api/_meta.json

```diff
@@ -5,9 +5,7 @@
     "newWindow": true
   },
   "http-api": {
-    "title": "HTTP API Reference",
-    "href": "https://www.postman.com/authzed/workspace/spicedb/overview",
-    "newWindow": true
+    "title": "HTTP API Reference"
```

Comment:
Should we make it clear that this is the SpiceDB HTTP API? Now that we have Cloud API around, which is also HTTP, the titles seem confusing. Proposal:
- SpiceDB gRPC API Reference
- SpiceDB HTTP API Reference
- Authzed Cloud HTTP API Reference